src/js/view/view.js

```diff
@@ -802,7 +802,7 @@ define([
         this.resize = function(width, height) {
             var resetAspectMode = true;
             _resize(width, height, resetAspectMode);
-            _responsiveListener();
+            _setContainerDimensions();
```

Comment:
Hmm, yes, though we want to execute _setContainerDimension immediately so we can update container sizes right away.  I think I can cancel any pending RAF calls in _setContainerDimensions for safety.